modules/gdscript/tests/scripts/parser/errors/class_name_after_annotation.gd

```diff
@@ -1,6 +1,6 @@
-# Error here. `class_name` should be used *before* annotations, not after.
+# Error here. `class_name` should be used *before* annotations, not after (except @tool).
 @icon("res://path/to/optional/icon.svg")
 class_name HelloWorld
 
 func test():
-	pass
+    pass
```

Comment:
Not important here but we seem to have some inconsistency between tabs and spaces in tests:
```
$ rg $"\t" | wc -l
2531
$ rg $"    " | wc -l
224
```

(Some of it may be intentional to actually test parsing different indentations, but probably not all.)